pynance/textimporter.py

```diff
@@ -108,6 +146,56 @@ def __init__(self,
         self.formatters = formatters
         self.skiprows = skiprows
         self.encoding = encoding
+        self.total_balance_re_pattern = total_balance_re_pattern
+
+    def read_total_balance(self, filepath_or_buffer):
+        """
+        Parses a filepath or buffer for a regex given in
+        total_balance_re_pattern and formats it into a number
+
+        PARAMS:
+        -------
+        filepath_or_buffer : str, pathlib.Path, py._path.local.LocalPath or
```

Comment:
I was a bit confused by the `str` here and thought that you could supply a raw string with the CSV content. Turns out that the string is interpreted as containing the path (am I right?). Please add a short note that `str` needs to contain the path to a file.

Reply:
I think you're right. You can turn a string into a buffer using `io.StringIO`, as I did in the tests - But if you just give a string, it will assume it's a path. I will make this clearer